inginious/frontend/pages/course_admin/submissions_data.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+from flask import request
+from bson.json_util import dumps
+
+from inginious.frontend.pages.course_admin.api_auth import DataAPIPage
+
+class SubmissionsEndpoint(DataAPIPage):
+    """ Endpoint to retrieve submission data """
+
+    def GET(self, courseid):
+        """ GET request """
+        courseid = self.verify()
+
+        # DB request parameters
+        params = {"courseid": courseid}
+        if "taskid" in request.args:
```

Comment:
Some safety checks might be interesting, as well as, at least, the same level of request convenience as the existing GUI.

If a user only wants to fetch evaluation submissions, they should'nt filter the whole databases results in their own code.